### PR TITLE
change master to main for new repositories

### PR DESCRIPTION
**Pull request by Joeperdefloep**
## Overview
**TL;DR**
<One to two sentence description of the issue you are encountering or trying to solve.>
Change master with main in the training manual as with the [change](https://www.zdnet.com/article/github-to-replace-master-with-main-starting-next-month/)
<Link to related issue. Type `closes #RELATEDISSUENUMBER` to establish a link.>

### Questions
<If relevant, write a list of questions that you would like to discuss related to the changes that you have made.>

### Next Steps
<If incomplete, create a task list of items that are still being worked on within the Pull Request.>

### Review
<If complete, or ready for :eyes:, use @mentions for quick questions, specific feedback, and progress updates.>

### CODE REVIEW

**Review by SonyViao — APPROVED** [review]

**Comment by brianamarie** [comment]
Thank you, @Joeperdefloep and @SonyViao!
